lib/teamsnap/api.rb

```diff
@@ -73,6 +73,8 @@ def self.via(klass, method)
 
     def self.parse_error(resp)
       return "Object Not Found (404)" if resp.status == 404
+      return "Forbidden (403)" if resp.status == 403 && resp.body == ""
```

Comment:
404 doesn't check for `resp.body == ""`. 
What I mean is why don't we have this:
```
return "Forbidden (403)" if resp.status == 403 || resp.status == 403 && resp.body == ""
```
That way on any 403 it returns the proper string, not just a body that is empty. Or will the body _always_ be empty on a 403?